Dockerfile

```diff
@@ -8,7 +8,7 @@ ARG TARGETPLATFORM=${TARGETPLATFORM}
 ARG BUILDPLATFORM=${BUILDPLATFORM}
 
 # Ruby image to use for base image, change with [--build-arg RUBY_VERSION="3.2.3"]
-ARG RUBY_VERSION="3.2.3"
+ARG RUBY_VERSION="3.2.4"
```

Comment:
There are a few other "3.2.3" strings in this file (in comments). Not critical to update, but maybe confusing for people to see diff values?